ompi/mca/osc/ucx/osc_ucx.h

```diff
@@ -44,6 +45,16 @@ typedef struct ompi_osc_ucx_component {
 
 OMPI_DECLSPEC extern ompi_osc_ucx_component_t mca_osc_ucx_component;
 
+#define INCREMENT_OUTSTANDING_NB_OPS                                \
```

Comment:
These should be function macros (they could be a `static inline` function, too):

```suggestion
#define INCREMENT_OUTSTANDING_NB_OPS()                              \
```

Reply:
done